doc/classes/AudioEffectBandLimitFilter.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectBandLimitFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Band Limit Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect limits the frequences surrounding a range around the [code]Cutoff Hz[/code], and lets only the frequences out of this range pass.
```

Comment:
- remove the comma before "and"
- You can remove "The effect" at the beginning. Starting with "Limits the frequencies..." is fine.
- "only the frequencies outside this range"